modules/nw-ingress-converting-http-header-case.adoc

```diff
@@ -31,6 +31,23 @@ As a cluster administrator, you can convert the HTTP header case by entering the
 $ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"httpHeaders":{"headerNameCaseAdjustments":["Host"]}}}'
 ----
 +
+. Create a `Route` resource YAML file so that the annotation can be applied to the application.
++
+.Example of a route named `my-application`
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+ name: my-application
+ namespace: default
+spec:
+ to:
+  kind: Service
+  name: my-application
```

Comment:
I'm wondering about the difference of this `Route` object and the one down the procedure: 1) is it on purpose that they are from different namespaces? 2) is there a purpose in adding the annotation in 2 different ways (`oc annotate` in the yaml directly)?
I think I would prefer to see similar yamls for both to make the user experience smoother.

Reply:
i agree to Andrey, 
1. it should be in same name space
2. this was already created long back. i guess these are two different methods to configure. one by patching the controller and annotating the route and another by changing directly in the controller and route yaml files.

Even after the above both steps, the yaml files of the controller and route will be same for both methods.

Reply:
Thanks, Andrey and Melvin. I updated the namespace name.